modules/core_api/fsw/inc/cfe_version.h

```diff
@@ -27,21 +27,29 @@
 
 /* Development Build Macro Definitions */
 #define CFE_BUILD_NUMBER   434 /**< @brief Development: Number of development git commits since CFE_BUILD_BASELINE */
-#define CFE_BUILD_BASELINE "v7.0.0-rc4" /**< @brief Development: Reference git tag for build number */
+#define CFE_BUILD_BASELINE "equuleus-rc1" /**< @brief Development: Reference git tag for build number */
+#define CFE_BUILD_DEV_CYCLE 	"equuleus-rc2" /**< @brief Development: Release name for current development cycle */
+#define CFE_BUILD_TYPE          "Development Build" /**< @brief: Development: Type of build (Development Build or Release) */
+#define CFE_BUILD_CODENAME      "Equuleus" /**< @brief: Development: Code name for the current build */
 
 /* See \ref cfsversions for definitions */
 #define CFE_MAJOR_VERSION 6  /*!< @brief Major version number */
 #define CFE_MINOR_VERSION 7  /*!< @brief Minor version number */
 #define CFE_REVISION      99 /*!< @brief Revision version number. Value of 99 indicates a development version.*/
```

Comment:
CFE_REVISION should be 0 instead of MISSION_REV being 00. Can use to set version string to development vs release.